data/completion/etelpmoc.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
```

Comment:
complete.sh (sets up tab completion for snap-based things, and)  serialises the tab completion "request", and de-serialises the tab completion "response" into the result that's seen by the shell. etelpmoc.sh de-serialises the tab-completion request, performs the tab completion, and serialises its result into the tab completion response. etelpmoc is a weird, backwards, tab completion thing, which is why it's called complete, backwards.